components/webfield/NoteReviewStatus.js

```diff
@@ -124,7 +124,9 @@ const AcPcConsoleReviewerReminderModal = ({
   officialReviewName,
   setUpdateLastSent,
   submissionName,
+  messageSignature,
 }) => {
+  const { emailReplyTo, reviewersMessageSubmissionInvitationId } = useContext(WebFieldContext)
```

Comment:
Could we name this config var `messageReviewersInvitationId`? It would be good to establish a naming convention where these invitations are all called `message${group}InvitationId`

Reply:
What about `messageSubmissionReviewersInvitationId`? this invitation is per submission.

Reply:
Yes, that would be great.